### ISSUE TITLE
create_volume_meshes_whitegray

### ISSUE THREAD

**Issue by marcsous** [harmening/MRIsegmentation#1]
Looks like a missing function?

``
Unrecognized function or variable 'create_volume_meshes_whitegray'.
Error in start_segmentation (line 87)
create_volume_meshes_whitegray(input_img, maxvoxelvolume);
Error in segment_test (line 45)
start_segmentation(input_img); 
``

**Comment by harmening** [comment]
@marcsous yes, you're right. Sorry, the whitegray-function is part of my current research. Shouldn't be called by default. [Removed it](https://github.com/harmening/MRIsegmentation/commit/7f7839c0cda121b3250ebc363fe33836203a6059).
